version.h.in

```diff
@@ -4,5 +4,7 @@
 #define PNET_VERSION_MAJOR @PNET_VERSION_MAJOR@
 #define PNET_VERSION_MINOR @PNET_VERSION_MINOR@
 #define PNET_VERSION_PATCH @PNET_VERSION_PATCH@
+#define PNET_VERSION_GIT "@PNET_VERSION_GIT@"
```

Comment:
Semantic versioning supports build info like so: MAJOR.MINOR.PATH+BUILD. See https://semver.org/#spec-item-10

Would be nice if PNET_VERSION_PATCH included the build info in that way, if and only if p-net is built from a git repository.

Reply:
I made a string `PNET_VERSION` with `PNET_VERSION_MAJOR.PNET_VERSION_MINOR.PNET_VERSION_PATCH `if available, otherwise `PNET_VERSION_GIT`
The string is easy to use in different ports (linux, rt-kernel etc) without `STRINGIFY()`
Is that OK?